spacy/pipeline/spancat_exclusive.py

```diff
@@ -18,6 +18,12 @@
 @architectures = "spacy.SpanCategorizer.v1"
 scorer = {"@layers": "Softmax.v2"}
 
+[model.scorer.init_W]
+@initializers = "zero_init.v1"
+
+[model.scorer.init_b]
+@initializers = "zero_init.v1"
+
```

Comment:
@adrianeboyd It seems that using `Softmax.v2` directly causes an error in the `@initializers`, hence I put back these lines of code. 

When I inspected the output of `self._config` in [this line of code](https://github.com/explosion/spaCy/blob/master/spacy/language.py#L165), the `init_W` and `init_b` keys have functions within them (e.g. `<function zero_init at 0x7f46ffa68d30>,`), not their string names.

Reply:
Ah, now I remember some related discussions with users about this. Let me look at the details again...

Reply:
This should be fixed by https://github.com/explosion/thinc/pull/808, but would (obviously) require a new release of thinc and updated thinc requirements in spacy.

Reply:
Ok sure, I'll remove the `init_*` parameters for now and wait for the new thinc release

Reply:
Let's see if this works...

Reply:
This part works now, at least.